include/nbl/asset/CCPUMeshPackerV2.h

```diff
@@ -69,6 +69,8 @@ bool CCPUMeshPackerV2<MDIStructType>::commit(IMeshPackerBase::PackedMeshBufferDa
         size_t batchFirstIdx = ramb.indexAllocationOffset;
         size_t verticesAddedCnt = 0u;
 
+        _NBL_DEBUG_BREAK_IF(std::distance(mbBegin, it) == 100u || std::distance(mbBegin, it) == 150u);
```

Comment:
soem debugging stuff? should be there probably?

Reply:
yep, its gone